source/access/mfa_guide.rst

```diff
@@ -98,7 +80,12 @@ together with PuTTY. Make sure you install version 0.78 or later.
 
 Pageant stores the certificates together with your private SSH keys, which allows you
 to use VSC facilities whenever you are prompted for your identity.
-To setup your Pageant, please refer to :ref:`Using Pageant <using Pageant>`.
+To setup your Pageant, please refer to :ref:`Using Pageant <using Pageant>` in
+the :ref:`mfa quick start`, or go to the respecitive section further below :ref:`mfa_client_config`.
+Reminder: in order to store the session SSH certificate make sure Pageant
+is already up and running in the background before attempting a conneciton with Putty.
+
+.. _mfa_agent_nix:
```

Comment:
In line 120, there is this sentence hanging in the air, and it has a very wrong formatting:

"(to kill the agent use ``eval "$(ssh-agent -k)"``)"

Reply:
indeed, removed.